src/cdk/collections/selection.ts

```diff
@@ -34,12 +34,13 @@ export class SelectionModel<T> {
   }
 
   /** Event emitted when the value has changed. */
-  onChange: Subject<SelectionChange<T>> | null = this._emitChanges ? new Subject() : null;
+  change: Subject<SelectionChange<T>> | null = this._emitChanges ? new Subject() : null;
```

Comment:
Is this a breaking change?

Reply:
Actually, I've already made this change in this PR and deprecated it - #8286 - I'm going to revert this back to the original name and we can consolidate on that merge :)